From f39d15b5180bb2534ad2fbd45b4b3d825be9f004 Mon Sep 17 00:00:00 2001
From: Arnaud Pouliquen <arnaud.pouliquen@foss.st.com>
Date: Fri, 7 Oct 2022 10:51:28 +0200
Subject: [PATCH 0366/1141] arm64: dts: st: Add sysconf for the Cortex M33 boot
 on stm32mp251

To be able to set the Cortex-M33 vector table address, the
stm32-rproc driver needs access to this subsystem.

Signed-off-by: Arnaud Pouliquen <arnaud.pouliquen@foss.st.com>
Change-Id: I6c0b61f8c7a33f75835d4ec7e106832979e862af
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/313004
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Tested-by: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
Reviewed-by: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
Domain-Review: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
---
 arch/arm64/boot/dts/st/stm32mp251.dtsi | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm64/boot/dts/st/stm32mp251.dtsi b/arch/arm64/boot/dts/st/stm32mp251.dtsi
index 4ee579a0eb0b..5f9c773b4f50 100644
--- a/arch/arm64/boot/dts/st/stm32mp251.dtsi
+++ b/arch/arm64/boot/dts/st/stm32mp251.dtsi
@@ -2384,6 +2384,12 @@ nand-controller@4,0 {
 				status = "disabled";
 			};
 		};
+
+		a35ss_syscfg: syscon@48802000  {
+			compatible = "st,stm32mp25-a35ss-syscfg", "syscon";
+			reg = <0x48802000 0xac>;
+			status = "disabled";
+		};
 	};
 
 	mlahb: ahb {
-- 
2.39.2

